packages/system-tests/assets/sfdx-simple/.sfdx/tools/testresults/apex/test-result-707xx00000000BG.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Had to add these files under `testresults` in order to run the tests. Let me know if you have a better idea for generating these for testing purposes.